### DISCUSSION TITLE
lunii v3

### DISCUSSION THREAD

**Discussion by AlexTraveylan** [jersou/studio-pack-generator#23] [General]
Hello, does your application work with version 3 of lunii which uses wifi? If not, do you have any leads?

**Comment by jersou** [comment]
Short answer in English: yes, SPG is compatible with the v3 but you have to use the Studio project for story transfers.

D'après tes repo git, tu parles français alors, je te réponds en FR ;-) : l'appli SPG est faite pour fonctionner avec l'appli Studio, si Studio est compatible avec les v3, du coup ça passe...
studio-pack-generator ne transmet pas d'histoires sur la Lunii, il faut voir le projet Studio pour les transferts !

J'ai suivi que de très loin, mais il semble que la communauté a réussi à faire fonctionner Studio avec la Lunii v3, il y a 2 discords pour en savoir + : 

- Le discord principal (FR) avec 10000 membres : https://monurl.ca/lunii.creations
- Le discord dédié à la v3 (FR) : https://discord.com/channels/1166815138645553172/1166815139450847263

**Reply by AlexTraveylan** [reply]
Je suis trop hipe, je vais tester immédiatement !

Merci énormement pour ta réponse !
